pkg/agent/openflow/client.go

```diff
@@ -480,6 +488,17 @@ func (c *client) UninstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcP
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
```

Comment:
@weiqiangt : BTW, I wonder why in InstallEndpointFlows() we pass the extra isIPv6 argument, but not just let net.ParseIP() return a net.IP and check it is v4 or v6 when necessary. Is it for some efficiency consideration?

Reply:
Yes, for efficiency consideration, otherwise we will create a small bytes slice each time we check. cc @ksamoray.

Reply:
But in this case, the IP is checked at most twice: one in endpointDNATFlow() and one in hairpinSNATFlow, and the latter is only for local Endpoint. Is it worth to optimize?

Reply:
Considering parameters passing is simple I thought it is OK. But a second thought is that the cost here is not that critical, I'm fine with the change also.

Reply:
As endpointDNATFlow() performs To4() anyway for v4, this argument has no importance as To4() is later applied again here
https://github.com/vmware-tanzu/antrea/blob/main/pkg/agent/openflow/pipeline.go#L1830

Reply:
Thanks for the info guys! So, let us remove the argument then.